_pytest/config.py

```diff
@@ -669,10 +674,13 @@ def _addoption_instance(self, option, shortupper=False):
 
 
 class MyOptionParser(argparse.ArgumentParser):
-    def __init__(self, parser):
+    def __init__(self, parser, extra_info={}):
```

Comment:
Likely not an issue but: 
http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments